spec/models/coordinates_spec.rb

```diff
@@ -4,6 +4,28 @@
   let(:service) { MetadataPresenter::Service.new(latest_metadata) }
   let(:uuid) { 'ad011e6b-5926-42f8-8b7c-668558850c52' } # Page N
 
+  describe 'initialising the branch spacers' do
+    context 'when branching point does not have OR conditions' do
+      let(:latest_metadata) { metadata_fixture(:branching_12) }
```

Comment:
if this is the only place branching 12 is used, and you only need it because it does not have an OR condition, then any of the branching fixtures except `branching` would do

Reply:
Happy to remove branching fixture 12 as it is only used here, although none of the other fixtures have revealed this bug in the past and I would like to specifically test this bug. I can make it work with some manipulation of fixture 10 but it would mean a bit of test setup, I can give it a go.

Reply:
Turned out fixture 10 was fine to use without adjustment as it had a similar set up to fixture 12 🎉 
Have now done this.